README.md

```diff
@@ -1,53 +1,96 @@
+---
 
-# How to use?
+## Is this fork up-to-date with original Vencord?
+Yes, it usually is, but we don't update this fork immediatley after the original Vencord is updated. You'll have to update it manually. We release a new version when there are necessary changes.
 
-If you want to use these plugins in your Discord client, please follow the instructions below or watch [this tutorial](https://drive.google.com/file/d/1zlWv4t14ORqy7QuF_Sup7DLkf7pAgS90/view?usp=sharing).
+---
 
+# Dependencies
 1. Download the latest version of Node.js from [here](https://nodejs.org/en/download/current).
-2. Download Git from [here](https://git-scm.com/).
+2. Download Git from [here](https://git-scm.com/download).
 3. Open a terminal and run the following commands:
 
-> [!IMPORTANT]  
-> Do not run any of these commands other than `npm install -g pnpm` as administrator, since you can cause updating the client to break.
+# Installing
+You can watch [this](https://drive.google.com/file/d/1zlWv4t14ORqy7QuF_Sup7DLkf7pAgS90/view?usp=sharing) tutorial to make it more easy.
+
+---
+
+Install `pnpm`:
+
+> [!IMPORTANT]
+> This next command may need to be run as admin/root depending on your system, and you may need to close and reopen your terminal for `pnpm` to be in your PATH.
+
+
+```shell
+npm i -g pnpm
+```
+
+---
+
+> [!IMPORTANT]
+> Make sure you aren't using an admin/root terminal from here onwards. It **will** mess up your Discord/Vencord instance and you **will** most likely have to reinstall.
 
-```bash
-npm install -g pnpm
-git clone https://github.com/Tolga1452/Vencord.git
+Clone Vencord:
+
+```shell
+git clone https://github.com/Vendicated/Vencord
 cd Vencord
+```
+---
+
+Install the dependencies needed:
+
+```shell
 pnpm install --frozen-lockfile
+```
+---
+
+Build this fork of Vencord:
+
+```shell
 pnpm build
-pnpm inject
 ```
+---
+
+Inject Vencord into your client:
 
+> [!TIP]
+> If you want no errors about permissions, you can close your Discord client by exiting it via the tray. But, if you don't want to, that's fine. Either way works.
+
+```shell
+pnpm inject
+```
+---
 When you want to update your Discord client to the latest version of this fork, just run `npm run update`.
 
 # Plugins
 
-## Modified
+## Modified Plugins
 
 ### [ClientTheme](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/clientTheme>)
 
-- Added dynamic color support.
+- Added support for dynamic colour.
 
 ### [Experiments](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/experiments>)
 
-- Added staging toggle.
+- Added a toggle to switch the release channel to staging.
 
-## Added
+## Added PLugins
 
-- **[Become Catgirl](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/becomeCatgirl>):** Turns your messages into a catgirl message.
+- **[Become Catgirl](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/becomeCatgirl>):** Turns your messages into catgirl style.
```

Comment:
If you want to edit plugin descriptions, edit the actual ones instead of readme first
```suggestion
- **[Become Catgirl](<https://github.com/Tolga1452/Vencord/tree/main/src/plugins/becomeCatgirl>):** Turns your messages into a catgirl message.
```